vic/vic_run/include/vic_log.h

```diff
@@ -45,52 +45,86 @@
 #include <errno.h>
 #include <string.h>
 
-FILE *LOG_DEST;
-
-#ifdef NDEBUG
-#define debug(M, ...)
-#else
-#define debug(M, ...) fprintf(LOG_DEST, "[DEBUG] %s:%d: " M "\n", __FILE__, \
-                              __LINE__, ## __VA_ARGS__)
+// Set the log level
+// To turn off warning statments, set LOG_LVL >= 30
+// | Level     | Numeric value    |
+// |---------  |---------------   |
+// | ERROR     | Always Active    |
+// | WARNING   | < 30             |
+// | INFO      | < 20             |
+// | DEBUG     | < 10             |
+#ifndef LOG_LVL
+#define LOG_LVL 25
 #endif
 
+FILE *LOG_DEST;
+
 void finalize_logging(void);
 void get_current_datetime(char *cdt);
 void get_logname(const char *path, int id, char *filename);
 void initialize_log(void);
 void setup_logging(int id);
 
-// do not try to be smart and make this go away on NDEBUG, the _debug
-// here means that it just doesn't print a message, it still does the
-// check.  MKAY?
-#define check_debug(A, M, ...) if (!(A)) {debug(M, ## __VA_ARGS__); errno = 0; \
-                                          exit(1); }
-
+// Macros for logging
 #define clean_errno() (errno == 0 ? "None" : strerror(errno))
 #define clean_ncerrno(e) (nc_strerror(e))
 
+// Debug Level
+#if LOG_LVL >= 10
```

Comment:
Confusing, the comment take says

`| DEBUG | < 10   |`

but the logic here says

```C
// Debug Level
#if LOG_LVL >= 10
```

Suggest making the comment and consistent, not just in the values but also in the logic ... same for the one below